### DISCUSSION TITLE
Feature Request: Add --force option

### DISCUSSION THREAD

**Discussion by Wizyma** [BetaHuhn/deploy-to-vercel-action#162] [Ideas] [answered]
## Context
Hi 👋  ! 
First of all, super cool action, would it be possible to add the `--force` option to the action to disable the vercel cache ? 
By default it wouldn't be set so it will always hit the cache if available, but sometimes we do not want to directly hit it and just to bypass it... ! 

## Changes
In the vercel command line file add a new variable that can be passed through the action, maybe called **FORCE** ? to refer to the `--force` option from the vercel CLI ? 

## More info
cf the vercel cli [docs](https://vercel.com/docs/cli#introduction/global-options/force)

Thank you !

**Comment by BetaHuhn** [comment, marked as answer]
Hi @Wizyma, thanks for the kind words!

It would be possible to add support for the `--force` option, but I don't have the time at the moment to add it myself. Feel free to create a PR though!

**Reply by Wizyma** [reply]
i made a PR, seems like its been a while since the last pkg-lock.json update :D !
